docs/kvbc/img/BlockMerkleCategoryAdd.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe it makes sense to put numbers on the arrows, to be more explicit about the flow order.

Reply:
Leaving for now out of expediency.